index.html

```diff
@@ -0,0 +1,126 @@
+            <a href="#" class="twitter"><img src="./images/twitter.svg" alt="twitter" /></a>
+            <a href="#" class="linkedin"><img src="./images/linkdin.svg" alt="linkdin" /></a>
+            <a href="#" class="english">English</a>
+            <a href="#" class="page">My Page</a>
+            <a href="#" class="logout">Logout</a>
+        </div>
+        <nav>
+            <div class="nav-bar">
+                <img class="humbuger" src="./images/menu.png" alt="Menu" />
+                <ul class="menu-item-hidden">
+                    <li class="cancel"><img src="./images/xicon.png" alt=""></li>
+                    <li><a href="./about/about.html" class="about">About</a></li>
+                    <li><a href="#Program" class="nav-list">Program</a></li>
+                    <li><a href="#join" class="nav-list">Join</a></li>
+                    <li><a href="#sponsor" class="nav-list">Sponsor</a></li>
+                    <li><a href="#news" class="nav-list">News</a></li>
+                    <li><a href="#campain" class="campain">CC Campain</a></li>
+                </ul>
+            </div>
+            <div class="nav-menu">
+                <a href="#home-page" class="logo">
+                    <img src="./images/logo.jpg" alt="logo"></a>
+                <ul class="menu-items">
+                    <li class="cancel"><img src="./images/xicon.png" alt=""></li>
+                    <li><a href="./about/about.html" class="about">About</a></li>
+                    <li><a href="#Program" class="nav-list">Program</a></li>
+                    <li><a href="#join" class="nav-list">Join</a></li>
+                    <li><a href="#sponsor" class="nav-list">Sponsor</a></li>
+                    <li><a href="#news" class="nav-list">News</a></li>
+                    <li><a href="#campain" class="campain">CC Campain</a></li>
+                </ul>
+            </div>
+            <div class="home-page">
+                <div class="title">
+                    <p class="greeting"><strong>Hello World!</strong></p>
+                    <h1>Artifitial Intelligence Webinar 2022</h1>
+                </div>
+                <div class="disc">
+                    <p>The life we are living today and how is the Artificial intelligence is evolving to our daily life
+                    </p>
+                </div>
+                <div class="time">
+                    <h4>2022.10.15(Mon.)~ 16(Tue.)</h4>
+                    <p>@Rwanda Convertion center of Kigali</p>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <section class="activities">
+        <h3> Main Program</h3>
+        <div class="allparts">
+            <div class="part">
+                <img src="./images/topic.png" alt="Menu" />
+                <p class="type">Topics</p>
+                <p class="explanation">Take look on the topics and ideas will be sharing in the Artificial Intelligence
+                    webinar</p>
+            </div>
```

Comment:
- [ ] Please kindly transform the `<div class="part">` cards into a more semantic element like an `<article>` to make the code cleaner and easier to navigate and read.

Reply:
resolved 👍